src/contrib/testkits/Akka.TestKit.Xunit2/Attributes/LocalFactAttribute.cs

```diff
@@ -0,0 +1,51 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="LocalFactAttribute.cs" company="Akka.NET Project">
+//     Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using Xunit;
+
+namespace Akka.TestKit.Xunit2.Attributes
+{
+    /// <summary>
+    /// <para>
+    /// This custom XUnit Fact attribute will skip unit tests if the environment variable
+    /// "XUNIT_SKIP_LOCAL_FACTS" exists and is set to the string "true"
```

Comment:
Do we want to distribute this as part of the testkit for XUnit generally? Or just internally for Akka.NET use? I'm open to either - just want to make sure I know what you want to do @Arkatufus

Reply:
I figured that its a fairly harmless addition, nothing will happen if you use it unless you specifically set the environment variable. We might as well make it a publicly available attribute.

Reply:
ok, sounds good to me.